style.css

```diff
@@ -0,0 +1,144 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+  .not-found {
+    height: 25px;
+    margin-left: 24px;
+    margin-top: 20px;
+    position: absolute;
+    width: 133px;
+  }
+
+  .letter-not-found {
+    font-family: 'Inconsolata', monospace;
+    font-style: normal;
+    font-size: 18px;
+    font-weight: 550;
+    height: 25px;
+    letter-spacing: -0.090em;
+    text-transform: uppercase;
+    width: 133px;
+  }
+
+  .scarecrow {
+    background: url("./scarecrow.png") center/cover no-repeat;
+    height: 220.35px;
+    margin-left: 44px;
+    margin-top: 109px;
+    position: absolute;
+    width: 286.83px;
+  }
+
+  .bad-news-title {
+    font-family: Space Mono;
```

Comment:
You're declaring this `font-family` in many places that are not needed, you should import it just in the outer container of the elements that are using it. You should be using only three font-family definitions, as in the picture. The elements inside the container should inherit the font of their parents.
![image](https://user-images.githubusercontent.com/29681222/114285597-05af4100-9a1e-11eb-975a-4ce80583f9fa.png)